docs/source/etl/merge.rst

```diff
@@ -0,0 +1,94 @@
+.. _merge-datasets:
+
+========================
+Merging Variant Datasets
+========================
+
+.. invisible-code-block:: python
+
+    import glow
+    glow.register(spark)
+    path1 = 'test-data/vcf-merge/HG00096.vcf.bgz'
+    path2 = 'test-data/vcf-merge/HG00097.vcf.bgz'
+
+Users frequently want to merge multiple VCF, BGEN, or Plink files from non-overlapping sample sets
+into a multi-sample dataset. You can use Glow and Spark to perform this merge operation with a few
+lines of code. In these examples, we will read from VCF files, but the same logic works
+on DataFrames backed by other file formats.
+
+First, you need to read the VCF files into a single Spark DataFrame:
+
+.. code-block:: python
+
+  from pyspark.sql.functions import *
+
+  df = spark.read.format('vcf').load([path1, path2])
+  df.select('contigName', 'start', col('genotypes').sampleId)\
+    .orderBy('contigName','start').show()
+
+  # Alternatively, you can use the "union" DataFrame method
+  df1 = spark.read.format('vcf').load(path1)
+  df2 = spark.read.format('vcf').load(path2)
+  df = df1.union(df2)
```

Comment:
What happens if the VCFs have different header lines/resulting schemas? I think `union` is less tolerant, right?